### PR TITLE
all return types are displayed for public members

### PR DESCRIPTION
**Pull request by robynHM**
- [ ] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -20,3 +20,5 @@ addSbtPlugin("org.irundaia.sbt" % "sbt-sassify" % "1.5.1")
 addSbtPlugin("net.ground5hark.sbt" % "sbt-concat" % "0.2.0")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-uglify" % "2.0.0")
+
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.10.4")
```
Comment (brendanstraughan) [inline project/plugins.sbt:24]: do we need to get approval for plugins? If so I guess we can always remove this plugin now that we have added all the type annotations

Comment (brendanstraughan) [reply project/plugins.sbt:24]: cc @vivekhmrc

Comment (vivekhmrc) [reply project/plugins.sbt:24]: @brendanstraughan / @robynHM I checked and it is being used in a lot other repositories so I think we are good :-)

**Review by brendanstraughan — APPROVED** [review]

**Review by sergi-hmrc — APPROVED** [review]
excellent job @robynHM! Thanks for adding scalafix :)

**Pull request merged**